compiler/src/dotty/tools/dotc/tastyreflect/SymbolOpsImpl.scala

```diff
@@ -80,6 +80,9 @@ trait SymbolOpsImpl extends scala.tasty.reflect.SymbolOps with CoreImpl {
       symbol.topLevelClass.asClass.isDefinedInCurrentRun
     }
 
+    def isType(implicit ctx: Context): Boolean = symbol.isType
```

Comment:
IMHO, instead of `IsTypeSymbol`, a method like `Symbol.isClass` is more useful for meta-programming, as it is the case in the compiler. Currently, I feel that the API & its types are too fine-grained, which creates unnecessary complexity for macro authors. As a macro author, I'd like to know fewer types and concepts, otherwise the cognitive load is high.